include/tvm/target/se_scope.h

```diff
@@ -170,7 +170,7 @@ class SEScopeNode : public AttrsNode<SEScopeNode> {
    *
    * kInvalidDeviceType denotes unconstrained.
    */
-  int device_type_int;
+  int /* actually DLDeviceType */ device_type_int;
```

Comment:
Should we make these private in C++?

Reply:
[Addressing these comments in sequel #9613]

Done.